### DISCUSSION TITLE
Welcome to C-Language Discussions!

### DISCUSSION THREAD

**Discussion by Ajit745** [Ajit745/C-Language#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ajit745** [comment]
hii

**Comment by Ajit745** [comment]
hii

**Comment by sagarverma-exe** [comment]
hii'

**Comment by Ronitj17121** [comment]
HII

**Comment by Ajit745** [comment]
https://www.programiz.com/c-programming/examples

**Reply by rjtheking1** [reply]
https://aptitude-test.com/

**Comment by rjtheking1** [comment]
https://aptitude-test.com/

**Comment by sagarverma-exe** [comment]
https://examrw8.rnwmultimedia.edu.in/student/dashboard
